src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -0,0 +1,59 @@
+package frc.robot.subsystems.elevator;
+
+import static frc.robot.subsystems.elevator.ElevatorConstants.*;
+
+import com.ctre.phoenix6.signals.ControlModeValue;
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class Elevator extends SubsystemBase {
+
+    private static Elevator INSTANCE;
+    private final ElevatorInputs inputs = new ElevatorInputs();
+    private final ElevatorIO io;
+    private final Mechanism2d mechanism2d = new Mechanism2d(MECHANISM_WIDTH, MECHANISM_HEIGHT);
+    private final MechanismRoot2d root = mechanism2d.getRoot("Elevator", ROOT_X, ROOT_Y);
```

Comment:
Why is that needed?

Reply:
d72f43b1c29abae9400cba633f56020634a13317

Reply:
You shouldn't have to log the mechanism to SmartDashboard, provided you log it using AdvantageKit. You do need to update the mechanism angle/height

Reply:
You shouldn't have to log the mechanism to SmartDashboard, provided you log it using AdvantageKit. You do need to update the mechanism angle/height